src/circuit/gadget/poseidon.rs

```diff
@@ -67,7 +67,7 @@ pub struct Word<
     const T: usize,
     const RATE: usize,
 > {
-    inner: PoseidonChip::Word,
+    pub inner: PoseidonChip::Word,
```

Comment:
Hmm, this should probably be a crate-public getter (which could then be made public later if this chip is extracted).